Makefile

```diff
@@ -5,17 +5,23 @@ V := @
 BINDIR := ./bin
 SY_CRI := $(BINDIR)/sycri
 SY_CRI_SELINUX := $(BINDIR)/sycri-selinux
+SECCOMP := $(shell echo '\#include <seccomp.h>\nint main() { }' | gcc -x c -o /dev/null -lseccomp - >/dev/null 2>&1; echo $$?)
+BUILD_TAGS :=
+ifeq ($(SECCOMP), 0)
+	BUILD_TAGS += seccomp
+endif
+
 
 .PHONY: build
-build: clean $(SY_CRI) $(SY_CRI_SELINUX)
+build: $(SY_CRI) $(SY_CRI_SELINUX)
 
 $(SY_CRI):
 	@echo " GO" $@
-	$(V)export GOOS=linux && go build -o $(SY_CRI) ./cmd/server
+	$(V)export GOOS=linux && go build -tags "$(BUILD_TAGS)" -o $(SY_CRI) ./cmd/server
 
 $(SY_CRI_SELINUX):
 	@echo " GO" $@
-	$(V)export GOOS=linux && go build -tags 'selinux' -o $(SY_CRI_SELINUX) ./cmd/server
+	$(V)export GOOS=linux && go build -tags "selinux $(BUILD_TAGS)" -o $(SY_CRI_SELINUX) ./cmd/server
```

Comment:
Why there is a separate binary for SELinux ?

Reply:
I decided to separate them because it turned out AppArmor & SELinux are mutually exclusive. It just hit me that it is not related to build tags at all and we can combine them. Thanks for your notice, I will remove that